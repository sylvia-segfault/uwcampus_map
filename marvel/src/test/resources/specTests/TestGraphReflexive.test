# Test BFS algorithm on a graph that contains self-loops

LoadGraph graph biggerData.tsv
# Add a reflexive edge
AddEdge graph B B BB
FindPath graph A E
