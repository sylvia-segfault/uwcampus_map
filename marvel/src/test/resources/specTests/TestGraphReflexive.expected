# Test BFS algorithm on a graph that contains self-loops

loaded graph graph
# Add a reflexive edge
added edge BB from B to B in graph
path from A to E:
A to B via AB
B to D via BD
D to E via DE